pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1021,6 +1006,7 @@ func TestHighPriorityFlushUnschedulableQLeftover(t *testing.T) {
 	q.AddUnschedulableIfNotPresent(q.newQueuedPodInfo(&highPod), q.SchedulingCycle())
 	q.AddUnschedulableIfNotPresent(q.newQueuedPodInfo(&midPod), q.SchedulingCycle())
 	c.Step(unschedulableQTimeInterval + time.Second)
```

Comment:
this is not necessary now since flush is called explicitly, right?

Reply:
It's still needed as the flush logic is depending on the time difference:

https://github.com/kubernetes/kubernetes/blob/27f06962d84868be93ead72d0f14eb5922020c4d/pkg/scheduler/internal/queue/scheduling_queue.go#L355-L362

A nit is to update `xyz + time.Second` to `xyz + time.Microsecond` in all places. But it's trivial as they're just fake clocks.